### PR TITLE
[template] drop 'intent to implement' language.

### LINKED ISSUE #436: [template] Rethink the "intents to implement" wording
The charter template includes "<em>Consider adding this clause if the Group does not intend to move to REC:</em> All new features should be supported by at least two intents to implement before being incorporated in the specification."

I see 2 problems here:

1. As @hober recently pointed out, not all browsers use intents at all. Even Chromium doesn't use the "Intent to Implement" anymore: it's now an "Intent to Prototype". I think the goal is relatively clear--this wants two implementers to state that they plan to implement the feature before it lands in a specification--but the wording isn't ideal.

2. I'm not sure this is actually the bar we want for living standards. The template's wording roughly matches the WHATWG's [working mode](https://whatwg.org/working-mode#additions), in requiring 2 implementers to say "we would like to implement this soon", but it's less than the W3C's current bar for [transitioning to PR](https://www.w3.org/2023/Process-20230612/#transition-pr) that there's adequate implementation experience. When a group doesn't intend to ever transition to PR because it doesn't want to freeze the specification for AC review, its charter should still say how the group will mark the features that have independent interoperable implementations. https://github.com/patcg/patwg-charter/pull/62 suggests one way to do this, but it's not the only possibility.

### PR DESCRIPTION
**Pull request by hober**
Fixes #436.

### CODE REVIEW

**Comment by hober** [comment]
@jyasskin

**Comment by hober** [comment]
see also patcg/patwg-charter#71

File: charter-template.html [hunk 1]
```diff
@@ -266,7 +266,7 @@ <h2>Success Criteria</h2>
     </p>
 	
     <!-- Relevance and momentum -->
-	  <p><span class='todo'>Consider adding this clause if the Group does not intend to move to REC:</span> All new features should be supported by at least two intents to implement before being incorporated in the specification.</p>
+	  <p><span class='todo'>Consider adding this clause if the Group does not intend to move to REC:</span> All new features should have expressions of interest from at least two potential implementors before being incorporated in the specification.</p>
```
Comment (jyasskin) [inline charter-template.html:269]: Just double-checking: Do you think "interest" is sufficient, or should this match the WHATWG working mode in requiring ["support"](https://whatwg.org/working-mode#additions)? I think I'll be happier about "interest" once there's also an indication at the CR phase of which features have 2 implementations, and I'm fine with merging this PR in the expectation that that text will arrive later.

**Review by jyasskin — APPROVED** [review]
Note that this only fixes the first half of #436.

File: charter-template.html (hunk 1 above)
Comment (hober) [reply charter-template.html:269]: I'd be fine with the WHATWG wording.

**Comment by AramZS** [comment]
👍

**Review by svgeesus — APPROVED** [review]
This is better wording. Fine to merge, but if people prefer other wording lets see that (so I will approve but not merge)

**Pull request merged**